### PR TITLE
Added Scarlet / Violet data for Pokemon Names / Pokemon Forms / Pokemon

### PR DESCRIPTION
**Pull request by giginet**
I added some info it's proceeded by #778 

The form changed Pokémon are not in the current data.

I updated followings:

- version_groups
- pokemon
- pokemon_names (Japanese/English)
- pokemon_forms
- pokemon_types
- pokemon_stats

Some column is missing

- pokemon.base_experience
- pokemon_names.genus
- pokemon_forms/pokemon.order

### CODE REVIEW

File: data/v2/csv/pokemon_species_names.csv [hunk 1]
```diff
@@ -9947,3 +9947,312 @@ pokemon_species_id,local_language_id,name,genus
...
+930,1,ウミトリオ,
+930,9,Wugtrio,
+930,11,ウミトリオ,
+931,1,ヘイラッシャ,
+931,9,Dondozo,
+931,11,ヘイラッシャ,
+932,1,ミガルーサ,
+932,9,Veluza,
+932,11,ミガルーサ,
+933,1,ナミイカル,
```
Comment (maiko-ando) [inline data/v2/csv/pokemon_species_names.csv]: @giginet 
ポケモン名Typoしてますね

File: data/v2/csv/pokemon_species_names.csv [hunk 2]
```diff
@@ -9947,3 +9947,312 @@ pokemon_species_id,local_language_id,name,genus
...
+962,1,ガケガニ,
+962,9,Klawf,
+962,11,ガケガニ,
+963,1,コジオ,
+963,9,Nacli,
+963,11,コジオ,
+964,1,ジオヅム,
+964,9,Naclstack,
+964,11,ジオヅム,
+965,1,キヨジオーン,
```
Comment (maiko-ando) [inline data/v2/csv/pokemon_species_names.csv]: @giginet 
キョジオーン

File: data/v2/csv/pokemon_species_names.csv [hunk 3]
```diff
@@ -9947,3 +9947,312 @@ pokemon_species_id,local_language_id,name,genus
...
+983,1,ハバタクカミ,
+983,9,Flutter Mane,
+983,11,ハバタクカミ,
+984,1,チヲハウハネ,
+984,9,Slither Wing,
+984,11,チヲハウハネ,
+985,1,トドロクツキ,
+985,9,Roaring Moon,
+985,11,トドロクツキ,
+986,1,テツノウダチ,
```
Comment (maiko-ando) [inline data/v2/csv/pokemon_species_names.csv]: @giginet 
テツノワダチ

**Comment by giginet** [comment]
gender_rate was wrong. I‘ll fix this tomorrow

**Comment by Naramsim** [comment]
Hi @giginet , thanks for the new data!

Could you also take a look at this: https://github.com/PokeAPI/pokeapi/pull/783#issuecomment-1337853382 ?

**Comment by giginet** [comment]
I miss understood, `gender_rate` is not included in this diff. So this looks good.
Could you review it? @Naramsim

File: data/v2/csv/pokemon.csv [hunk 4]
```diff
@@ -904,6 +904,109 @@ id,identifier,species_id,height,weight,base_experience,order,is_default
 903,sneasler,903,13,430,,,1
 904,overqwil,904,25,605,,,1
 905,enamorus-incarnate,905,16,480,,,1
+906,sprigatito,906,0.4,4.1,,906,1
```
Comment (Naramsim) [inline data/v2/csv/pokemon.csv]: Hi, `height` and `weight` use decimetres and hectograms. Moreover I don't think we support _floats`. So you just need to multiply the values by 10.

**Review by Naramsim — CHANGES_REQUESTED** [review]

File: data/v2/csv/pokemon.csv (hunk 4 above)
Comment (giginet) [reply data/v2/csv/pokemon.csv]: Fixed
https://github.com/PokeAPI/pokeapi/pull/793/commits/b3eebc600747cb2261d0884924dbe392b00a8bc2

**Comment by oneirocosm** [comment]
I've noticed that the current order of the Paldean species follows the initial data-mined order from Scarlet and Violet.  What is the plan for when the official order is eventually released (will probably happen when Pokemon Home updates)?  I suppose you could either:

1. Keep the current order and update the National Dex Numbers in pokemon_dex_numbers.csv
2. Update the current order to match the official national dex order

If you do go with 2, it may be possible to do some of that work now using [the other popular order](https://bulbapedia.bulbagarden.net/wiki/User:ScraftyIsTheBest/Tentative_Generation_IX_National_Pok%C3%A9dex_Numbers) that I've been seeing lately.  There is no guarantee that it will match, but I think it is much more likely to than the current ordering based off what I'm reading [here](https://www.smogon.com/forums/threads/scarlet-violet-leaks-thread-data-mechanics.3709515/page-377#post-9420325]) and [here](https://www.smogon.com/forums/members/slp32.579658/#profile-post-2452287).

**Comment by giginet** [comment]
@oneirocosm Make sense. 
I also worry about current internal data including missing numbers.
I guess this internal numbering might be changed in a future series.
It's important to decide how to number IDs in PokéAPI.

Could somebody be concerned about which way is better?

In the latter way, other following data must be migrated to new IDs.

**Comment by pokeapi-machine-user** [comment]
A [PokeAPI/api-data](https://github.com/PokeAPI/api-data) refresh has started. In 45 minutes the staging branch of [PokeAPI/api-data](https://github.com/PokeAPI/api-data/tree/staging) will be pushed with the new generated data. <br><br> The staging branch will be deployed in our [staging environment](https://staging.pokeapi.co/api/v2/) and you will be able to review the entire API. <br><br> A Pull Request ([master](https://github.com/PokeAPI/api-data/tree/master)<-[staging](https://github.com/PokeAPI/api-data/tree/staging)) will be also created at [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) and assigned to the PokeAPI Core team to be reviewed. If approved and merged new data will soon be available worldwide at [pokeapi.co](https://pokeapi.co/api/v2/).

**Comment by giginet** [comment]
@Naramsim How about these discussions? https://github.com/PokeAPI/pokeapi/pull/793#issuecomment-1369192812

Should we use current numbering?

**Comment by Naramsim** [comment]
no clue, I didn't have time to read it. Could you and @oneirocosm decide?

**Comment by giginet** [comment]
@Naramsim Those discussions are important for future compatibilities.
So they should be decided by a core maintainer. We are not maintainers.

If you don't have any concerns, I'll process the following PRs. However, I think it's tough to modify this data structure in the future.

**Comment by Naramsim** [comment]
What about setting the `order` as a `null`able field and leaving it empty? Until the official numbering is released?

**Comment by giginet** [comment]
Sounds good. I'll use the current numbering for the following PRs.

**Comment by lordkyuusei** [comment]
A promotional video came out a few hours ago with what seems like an official ID numbering for each Paldean Pokemon, if it helps: https://www.youtube.com/watch?v=pa08Y-fhfTI (starting at 6:36)

**Comment by Naramsim** [comment]
yes, other people in another issue are aware of that

**Pull request merged**